Azure-Samples/communication-services-ios-calling-hero
# Include mute indicator on display name when toggled

## Purpose

Include mute indicator on display name when toggled by either local or remote participants

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
- toggle mute / unmute in waiting room when start new call
- toggle mute / unmute in 1:1 group call then add / remove participants to see if your own mute status is correct
- toggle mute / unmute in 3+ participants
- for remote participants in active display, toggle their mute / unmute
- for remote participants not in active display, toggle their mute / unmute, then bring them into active display to see if their mute status is correct
- see if all participants have correct mute status if join another call
- any other scenarios not stated here
